data/game.py

```diff
@@ -314,6 +315,105 @@ def current_play_result(self):
             in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
         ):
             result += "_looking"
+        elif result == "field_out" and (
+            "flies"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_fly"
+        elif result == "field_out" and (
+            "grounds"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_ground"
+        elif result == "field_out" and (
+            "lines"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_line"
+        elif result == "field_out" and (
+            "pop"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_pop"
```

Comment:
I think it would make sense to turn this into a helper function called like `out_type`